instances.go

```diff
@@ -260,6 +292,35 @@ func (c *Client) SetAllowedNetworkInstancePolicy(ctx context.Context, enable boo
 	return err
 }
 
+// SetKeyAccessInstancePolicy updates the key create import access policy details associated with an instance.
+// For more information can refer to the Key Protect docs in the link below:
+// https://cloud.ibm.com/docs/key-protect?topic=key-protect-manage-keyCreateImportAccess
+func (c *Client) SetKeyAccessInstancePolicy(ctx context.Context, enable bool, attributes *Attributes) error {
+	policy := InstancePolicy{
+		PolicyType: KeyAccess,
+		PolicyData: PolicyData{
+			Enabled: &enable,
+		},
+	}
+
+	policy.PolicyData.Attributes = attributes
+
+	policyRequest := InstancePolicies{
+		Metadata: PoliciesMetadata{
+			CollectionType:   policyType,
+			NumberOfPolicies: 1,
+		},
+		Policies: []InstancePolicy{policy},
+	}
+
+	err := c.setInstancePolicy(ctx, KeyAccess, policyRequest)
+	if err != nil {
+		return err
+	}
+
+	return err
```

Comment:
It's easier to read and understand if you `return nil` here instead of an `err` with a `nil` value.

This feels funny.

Reply:
well its not really a big thing. 
It can be understood as at the end nil is returned because err will be nil.